grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc"
+
+service ConfigService {
+    rpc GetGlobalConfig (GetGlobalConfigRequest) returns (GlobalConfig);
+    rpc UpdateGlobalConfig (UpdateGlobalConfigRequest) returns (UpdateGlobalConfigResponse);
+    rpc CheckConfigGeneration (CheckConfigGenerationRequest) returns (CheckConfigGenerationResponse);
+}
+
+message GetGlobalConfigRequest {}
+
+message UpdateGlobalConfigRequest {
+    GlobalConfig config = 1;
+}
+
+message UpdateGlobalConfigResponse {
+    bool status = 1;
```

Comment:
It's better to define an enum for the status with Ok/Error to start with so we can extend it in the future. https://protobuf.dev/programming-guides/enum/